components/expenses/DeleteExpenseBtn.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import gql from 'graphql-tag';
+import { graphql } from 'react-apollo';
+import { FormattedMessage } from 'react-intl';
+
+import SmallButton from '../SmallButton';
+
+class DeleteExpenseBtn extends React.Component {
+  static propTypes = {
+    id: PropTypes.number.isRequired,
+    deleteExpense: PropTypes.func.isRequired,
+  };
+
+  constructor(props) {
+    super(props);
+    this.onClick = this.onClick.bind(this);
+  }
+
+  async onClick() {
+    const { id } = this.props;
+    const res = window.confirm('Do you want to delete this rejected expense?');
```

Comment:
We should internationalize this one, it's more tricky when you want to output a simple string but still feasible. You'll need to have `defineMessages` + `injectIntl`. See https://github.com/opencollective/opencollective-frontend/blob/17638090c5b00f2373aa6b0ad0f5239d90ae4da2/components/EditMembers.js#L146